lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.html

```diff
@@ -26,7 +26,6 @@
         *ngIf="showDropdownSiteList"
         class="full-width"
         (change)="siteChanged($event)"
-        [placeholder]="'NODE_SELECTOR.SELECT_LOCATION'"
```

Comment:
Did you discuss it with someone if there should not be placeholder anymore?

Reply:
@jatin2008 Please confirm with Shane if we can remove placeholder from this place

Reply:
yeah i confirmed with shane, and he told me too add Select a Library instead of Select Location as a placeholder